.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "4.7.0"
```

Comment:
Is this something we will need to remember to manually update, from now on?
If so, it should probably be mentioned in https://shaka-project.github.io/maintenance/shaka-player.html

Reply:
No, the tool updates it on each release.  You'll see it tweaked in each release PR from now on.

I don't love tracking the last release number in yet another JSON file, but it seems to be required by the newer versions of the tool.  (It used to be an option, and we had to use it a time or two to bootstrap things.)